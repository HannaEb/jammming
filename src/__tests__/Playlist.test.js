import Playlist from '../Components/Playlist/Playlist.js';
import renderer from 'react-test-renderer';

jest.mock('../Components/TrackList/TrackList.js', () => 'TrackList');

describe('Playlist', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<Playlist />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
