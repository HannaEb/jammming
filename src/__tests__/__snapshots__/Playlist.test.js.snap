// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playlist matches the snapshot 1`] = `
<div
  className="Playlist"
>
  <input
    defaultValue="New Playlist"
    onChange={[Function]}
  />
  <TrackList
    isRemoval={true}
  />
  <button
    className="Playlist-save"
  >
    SAVE TO SPOTIFY
  </button>
</div>
`;
